projects/ngx-quill/src/lib/quill-editor.component.spec.ts

```diff
@@ -1355,7 +1355,7 @@ describe('QuillEditor - base config', () => {
       jasmine.objectContaining({attrName: 'size', keyName: 'font-size', scope: 5, whitelist: ['14']}), true, true
     )
 
-    // eslint-disable-next-line @typescript-eslint/dot-notation
+     
```

Comment:
lines with only white spaces / trailing whitespaces should not be allowed the, too

Reply:
yes, just run `eslint --fix`, seems we need more `@stylistic/eslint-plugin` rules

Reply:
i think you already know, but maybe https://github.com/angular-eslint/angular-eslint/tree/main/packages/angular-eslint/src/configs is helping

Reply:
demo of angular-eslint merged ts rules and angular-ts rules, if you pefer merge them, we can handle it too